aws/provider.go

```diff
@@ -14,6 +17,15 @@ import (
 	tfaws "github.com/terraform-providers/terraform-provider-aws/aws"
 )
 
+// skippableErrors is a list of errors
+// which won't make Terracognita failed
+// but they will be printed on the output
+// they are based on the err.Message() content
+// of the AWS error
+var skippableErrors = []string{
+	"Unavailable Operation",
+}
```

Comment:
As what we want is to validate if this exists a `map[string]struct{}` it's the perfect structure.